pkgs/by-name/fl/flet-client-flutter/package.nix

```diff
@@ -19,7 +19,8 @@ flutter.buildFlutterApplication rec {
     rev = "v${version}";
     hash = "sha256-58SMashinWhWZQuo8LTh83eS6ES70jomlGmZtAP9fz4=";
   };
-  sourceRoot = "source/client";
+
+  sourceRoot = "${src.name}/client";
```

Comment:
This change shouldn't be part of the commit `python311Packages.flet: use shared src from flet-client-flutter as it's a monorepo`.